test/built-ins/TypedArray/prototype/sort/sort-tonumber.js

```diff
@@ -0,0 +1,39 @@
+// Copyright (C) 2018 Mozilla Corporation. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+esid: sec-%typedarray%.prototype.sort
+description: The result of compareFn is immediately passed through ToNumber
+info: |
+  22.2.3.26 %TypedArray%.prototype.sort ( comparefn )
+
+  ...
+  2. If comparefn is not undefined, then
+    a. Let v be ? ToNumber(? Call(comparefn, undefined, « x, y »)).
```

Comment:
I totally missed this change. Now I finally found this https://github.com/tc39/ecma262/pull/1000

I wish it could be easier to find these changes or make sure they are observed in Test262.